examples/tutorials/api/nested_sampling_Crab.py

```diff
@@ -0,0 +1,354 @@
+# stationary state. *Once convergence* is reached the successive elements
+# of the chain are samples of the target posterior distribution. However
+# the weakness of the MCMC approach lies in the *Once convergence* part.
+# Started far from the best likelihood region, the convergence time can be
+# long or never reached if the walkers fall in a local minima. The choice
+# of the initialization point can become critical for complex models with
+# a high number of dimensions and the ability of these walkers to escape a
+# local minimum or to accurately describe a complex likelihood space is
+# not guaranteed.
+#
+# Nested sampling approach
+# ~~~~~~~~~~~~~~~~~~~~~~~~
+#
+# | To overcome these issues, the nested sampling (NS) algorithm has
+#   gained traction in physics and astronomy. It is a Monte Carlo
+#   algorithm for computing an integral of the likelihood function over
+#   the prior model parameter space introduced in 2004 by John Skilling.
+#   The method performs this integral by evolving a collection of points
+#   through the parameter space (see recent reviews of `Ashton
+#   2022 <https://ui.adsabs.harvard.edu/abs/2022NRvMP...2...39A>`__, and
+#   of `Buchner 2023 <http://arxiv.org/abs/2101.09675>`__). Without going
+#   into too many details, one important specificity of the NS method is
+#   that it starts from the entire parameter space and evolves a
+#   collection of live points to map all minima (including multiple modes
+#   if any) whereas Markov Chain Monte Carlo methods require an
+#   initialization point and the walkers will explore the local
+#   likelihood. The ability of these walkers to escape a local minimum or
+#   to accurately describe a complex likelihood space is not guaranteed.
+#   This is a fundamental difference between MCMC (and Minuit) which will
+#   only ever probe the vicinity along their minimization paths and do not
+#   have an overview of the global likelihood landscape. The analysis
+#   using the NS framework is more CPU time consuming than a standard
+#   classical fit but provides the full posterior distribution for all
+#   parameters, which is out of reach with traditional fitting techniques
+#   (N*(N-1)/2 contour plots to generate). In addition it is more robust
+#   to the choice of initialization, requires less human intervention and
+#   is therefore readily integrated in pipeline analysis. In gammapy, we
+#   used the NS implementation of the UltraNest package, one of the
+#   leading package in Astronomy (already used in Cosmology and in
+#   X-rays).
+# | For more information on UltraNest see the docs here : `UltraNest
+#   docs <https://johannesbuchner.github.io/UltraNest/>`__
+# | And a nice visualization of the NS method : `sampling
+#   visulisation <https://johannesbuchner.github.io/UltraNest/method.html#visualisation>`__
+#
+# Reference :
+# ~~~~~~~~~~~
+#
+# -  Ultranest docs : https://johannesbuchner.github.io/UltraNest
```

Comment:
Should we put these references within our central reference file, such that it can be reused within some rst page? Any preference, @Astro-Kirsty ?